lib/humanized_id.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module HumanizedId
+  DEFAULT_GENERATION_LENGTH = 24
+  CHARACTERSET =
+    (('0'..'9').to_a + ('A'..'Z').to_a - %w(0 1 5 8 A B E I L O S U)).freeze
```

Comment:
For users of the gem, we may wish to note WHY this is the charset.
The choice of charset is entirely arbitrary.
The question is, what is 'easy to read' and what is 'vulgar'?
E.G. In this case, `0 1 5 8 O I L S B` are removed so that the result is easy to read, as the letters and numbers in that list are hard to tell apart.
The rest, the removed vowels, are dropped because English profanity is made of words with `U`, and general impolite words typically contain `A`, and occasionally `E`.